components/fieldgroups/weather/currentWeather.fieldgroup.json

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
Please rename this the file to "current-weather.schema.json". Also move it under shared folder (fieldgroups/shared)